packages/frontend/src/pages/SqlRunner.tsx

```diff
@@ -103,53 +77,34 @@ const SqlRunnerPage = () => {
-                            display: 'flex',
-                            flexDirection: 'column',
-                        }}
+        <PageBase>
+            <Sidebar title="SQL runner">
+                <H5 style={{ paddingLeft: 10 }}>Warehouse schema</H5>
+                <div style={{ overflowY: 'auto' }}>
+                    {isCatalogLoading ? (
+                        <SideBarLoadingState />
+                    ) : (
+                        <Tree
+                            contents={catalogTree}
+                            handleSelect={false}
+                            onNodeClick={handleNodeClick}
+                        />
+                    )}
+                </div>
+                <Tooltip2
+                    content="Currently we only display tables that are declared in the dbt project."
+                    className="missing-tables-info"
+                >
+                    <Callout
+                        intent="none"
+                        icon="info-sign"
+                        style={{ marginTop: 20 }}
                     >
-                        <div style={{ flex: 1 }}>
-                            <H3 style={{ marginBottom: 20 }}>SQL runner</H3>
-                            <Divider />
-                            <H5 style={{ marginTop: 20, paddingLeft: 10 }}>
-                                Warehouse schema
-                            </H5>
-                            <div style={{ overflowY: 'auto' }}>
-                                {isCatalogLoading ? (
-                                    <SideBarLoadingState />
-                                ) : (
-                                    <Tree
-                                        contents={catalogTree}
-                                        handleSelect={false}
-                                        onNodeClick={handleNodeClick}
-                                    />
-                                )}
-                            </div>
-                            <Tooltip2
-                                content="Currently we only display tables that are declared in the dbt project."
-                                className="missing-tables-info"
-                            >
-                                <Callout
-                                    intent="none"
-                                    icon="info-sign"
-                                    style={{ marginTop: 20 }}
-                                >
-                                    Tables missing?
-                                </Callout>
-                            </Tooltip2>
-                        </div>
-
-                        <AboutFooter />
-                    </div>
-                </Section>
+                        Tables missing?
+                    </Callout>
+                </Tooltip2>
             </Sidebar>
-            <ContentSection>
+            <Content>
                 <Section name={SectionName.EXPLORER_TOP_BUTTONS}>
```

Comment:
I guess this overrides the `PAGE_CONTENT` section? Maybe we can remove `PAGE_CONTENT` entirely?

Reply:
There are more elements inside <Content/> that are not inside this sub section.